Recipe_1.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="style.css">
+  <title>Pizza</title>
+</head>
+<body>
+  <header>
+    <nav>
+     <a href="index.html" class="logo">Happy<span1>Kitchen</span></a>
+    </nav>
+    <ul class="navbar">
+      <li><a href="Recipe_1.html" class="nav-list">Pi<span1>zza</span1> |</a></li>
+      <li><a href="Recipe_2.html" class="nav-list"><span1>Bur</span1>ger |</a></li>
+      <li><a href="Recipe_3.html" class="nav-list">Sa<span1>lad</span1></a></li>
+    </ul>
```

Comment:
Great work creating a nice navigation elements for easy site navigation. It's standard practice of HTML semantics to include all navigation links in your <code>nav</code> tag. Do you think you can replicate that too?